tests/Feature/Auth/RegistrationTest.php

```diff
@@ -1,47 +1,68 @@
 <?php
 
+declare(strict_types=1);
+
 namespace Tests\Feature\Auth;
 
+use App\Helpers\SignupHelper;
 use Illuminate\Foundation\Testing\DatabaseTransactions;
-use Laravel\Fortify\Features;
+use Illuminate\Http\Response;
 use Laravel\Jetstream\Jetstream;
-use PHPUnit\Framework\Attributes\Test;
+use Mockery;
 use Tests\TestCase;
 
 class RegistrationTest extends TestCase
 {
     use DatabaseTransactions;
 
-    #[Test]
-    public function registration_screen_can_be_rendered()
+    public function testAccessToRegistrationPage(): void
```

Comment:
Let's keep the new syntax
```suggestion
    #[Test]
    public function registration_screen_can_be_rendered(): void
```

Reply:
@asbiin, fixed